packages/sdks/output/qwik/src/blocks/section/section.tsx

```diff
@@ -0,0 +1,101 @@
+// GENERATED BY MITOSIS
+
+import { component$, useStore, h, Fragment, Slot } from "@builder.io/qwik";
+export const COMPONENT = {
+  "@type": "@builder.io/mitosis/component",
+  imports: [],
+  exports: {},
+  inputs: [],
+  meta: {},
+  refs: {},
+  state: {},
+  children: [
+    {
+      "@type": "@builder.io/mitosis/node",
+      name: "section",
+      meta: {},
+      scope: {},
+      properties: {},
+      bindings: {
+        _spread: {
+          code: "props.attributes",
+        },
+        style: {
+          code: "props.maxWidth && typeof props.maxWidth === 'number' ? {\n  maxWidth: props.maxWidth\n} : undefined",
+        },
+      },
+      children: [
+        {
+          "@type": "@builder.io/mitosis/node",
+          name: "div",
+          meta: {},
+          scope: {},
+          properties: {
+            _text: "\n      ",
+          },
+          bindings: {},
+          children: [],
+        },
+        {
+          "@type": "@builder.io/mitosis/node",
+          name: "div",
+          meta: {},
+          scope: {},
+          properties: {},
+          bindings: {
+            _text: {
+              code: "props.children",
+            },
+          },
+          children: [],
+        },
+        {
+          "@type": "@builder.io/mitosis/node",
+          name: "div",
+          meta: {},
+          scope: {},
+          properties: {
+            _text: "\n    ",
+          },
+          bindings: {},
+          children: [],
+        },
+      ],
+    },
+  ],
+  hooks: {},
+  context: {
+    get: {},
+    set: {},
+  },
+  name: "SectionComponent",
+  subComponents: [],
+  interfaces: [
+    "export interface SectionProps {\n  maxWidth?: number;\n  attributes?: any;\n  children?: any;\n  builderBlock?: any;\n}",
+  ],
+  propsTypeRef: "SectionProps",
+};
+export interface SectionProps {
+  maxWidth?: number;
+  attributes?: any;
+  children?: any;
+  builderBlock?: any;
+}
+const SectionComponent = component$((props: SectionProps) => {
+  const state = useStore({});
+  return (
+    <section
```

Comment:
realized we're not handling CSS currently from what I can tell. we prob would have the qwik generator generate whatever is idiomatic, IIRC that's a CSS file plus a `useScopedStyles$()` hook here?

Reply:
this has been added now.